core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -543,12 +529,15 @@ private BlockStoreLocation commitBlockInternal(long sessionId, long blockId, boo
     BlockStoreLocation loc = tempBlockMeta.getBlockLocation();
 
     // Heavy IO is guarded by block lock but not metadata lock. This may throw IOException.
-    FileUtils.move(srcPath, dstPath);
+    try {
+      FileUtils.move(srcPath, dstPath);
```

Comment:
This should probably also be changed to handle specific exception so here you don't need to catch IO exception.

Reply:
yes, I have a comment that I would have a follow-up PR once I figure out the Move API.